dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/AuditServiceImpl.java

```diff
@@ -115,11 +121,14 @@ public PageInfo<AuditDto> queryLogListPaging(User loginUser,
      */
     private AuditDto transformAuditLog(AuditLog auditLog) {
         AuditDto auditDto = new AuditDto();
-        String resourceType = AuditResourceType.of(auditLog.getResourceType()).getMsg();
-        auditDto.setResource(resourceType);
-        auditDto.setOperation(AuditOperationType.of(auditLog.getOperation()).getMsg());
+        AuditObjectType objectType = AuditObjectType.of(auditLog.getObjectType());
+        auditDto.setObjectType(objectType.getName());
+        auditDto.setObjectName(auditLog.getObjectName());
+        auditDto.setOperation(AuditOperationType.of(auditLog.getOperationType()).getName());
         auditDto.setUserName(auditLog.getUserName());
-        auditDto.setResourceName(auditLogMapper.queryResourceNameByType(resourceType, auditLog.getResourceId()));
+        auditDto.setLatency(String.format("%.2f", (double) auditLog.getLatency() / 1000));
```

Comment:
```suggestion
        auditDto.setLatency(String.format("%.2f", (double) auditLog.getLatency() / 1000));
```
It's better to use ms.

Reply:
done